src/range.rs

```diff
@@ -451,6 +544,72 @@ impl<V: Ord + Clone> Range<V> {
         Self { segments: output }.check_invariants()
     }
 
+    /// Return true if there can be no `V` so that `V` is contained in both `self` and `other`.
+    ///
+    /// Note that we don't know that set of all existing `V`s here, so we only check if the segments
+    /// are disjoint, not if no version is contained in both.
+    pub fn is_disjoint(&self, other: &Self) -> bool {
+        // The operation is symmetric
```

Comment:
Sounds like a nice property test to have regarding symmetry of the operator

Reply:
Good idea, done